deployments/helm/nvidia-device-plugin/templates/daemonset-mps-control-daemon.yml

```diff
@@ -169,6 +172,7 @@ spec:
             value: compute,utility
           securityContext:
             privileged: true
+            {{- include "mps-control-daemon.securityContext" . | nindent 10 }}
```

Comment:
Since we're already running the container as `privileged`, I don't think this is relevant here.